ezcater_rubocop.gemspec

```diff
@@ -34,6 +34,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency "bundler", "~> 1.15"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", "~> 3.0"
+  spec.add_development_dependency "pry-byebug"
```

Comment:
doesn't bundler complain when a version isn't specified? not a huge deal since it's pry and dev dependency

Reply:
Nope... it just always pulls the latest when you update:
![image](https://user-images.githubusercontent.com/7584015/31130202-bcf8a7f6-a824-11e7-9d40-5048d2b489af.png)

Reply:
To add to that, for a gem the `Gemfile.lock` is not committed. It will be generated and used locally, but developers working on the gem may use different versions unless you constrain them here.

Reply:
I'm ok with that for `pry`.... if you guys disagree, I'm happy to lock it to a version, but until there are issues with `pry`, I don't see any reason to force people onto a particular version. Let me know thoughts.

Reply:
I'm fine with no version restriction on pry.

Reply:
same